frontend/src/pages/BadgesPage.js

```diff
@@ -95,37 +95,19 @@ class BadgesPage extends Component {
   render() {
     return (
       <div className="profile">
-        <Container fluid={true}>
+        <Container>
```

Comment:
Is removing fluid intentional?

Reply:
yea it was just so the container was a little smaller and not pushed up against the sides of the page